tests/comfy_cli/test_tracking.py

```
@@ -0,0 +1,104 @@
+        assert kwargs["properties"]["k"] == "v"
+        assert "cli_version" in kwargs["properties"]
+        assert "tracing_id" in kwargs["properties"]
+
+    def test_properties_default_to_empty_dict(self, tracking_module):
+        tracking_module.config_manager.set(constants.CONFIG_KEY_ENABLE_TRACKING, "True")
+        tracking_module.track_event("some_event")
+        tracking_module.mp.track.assert_called_once()
+        _, kwargs = tracking_module.mp.track.call_args
+        assert set(kwargs["properties"].keys()) == {"cli_version", "tracing_id"}
+
+    def test_swallows_mixpanel_errors(self, tracking_module):
```

Pylint (4.0.5)

[convention] Missing function or method docstring

(C0116)